src/main/java/com/capitalone/dashboard/collector/GitHubCollectorTask.java

```diff
@@ -128,13 +130,6 @@ public GitHubCollector getCollector() {
         uniqueOptions.put(GitHubRepo.BRANCH, "");
         protoType.setUniqueFields(uniqueOptions);
 
-        if (existingCollector != null) {
-            protoType.setLastPrivateRepoCollectionTimestamp(existingCollector.getLastPrivateRepoCollectionTimestamp());
-            protoType.setLastCleanUpTimestamp(existingCollector.getLastCleanUpTimestamp());
-            protoType.setLastExecutionRecordCount(existingCollector.getLastExecutionRecordCount());
-            protoType.setLatestProcessedEventId((existingCollector).getLatestProcessedEventId());
-            protoType.setLatestProcessedEventTimestamp((existingCollector).getLatestProcessedEventTimestamp());
-        }
```

Comment:
Do we not need these fields in protoType?

Reply:
no need as we are returning the collector as soon as we find it . please check few lines above